maintenance/ebpf/ebpf-and-eks.md

```diff
@@ -80,35 +79,33 @@ container-optimised OS with an emphasis on security; it has a version of the ker
         - arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
   EOF
   ```
-
-* Install {{site.prodname}} using the following manifest from the AWS VPC CNI project:
-  ```bash
-  kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/ae02a103b091f38b0aafd0ff6dd0e8f611cf9e67/config/master/calico.yaml
+* Install the Tigera operator:
   ```
-  
-  > **Note**: Due to Bottlerocket's read-only file system, it is not possible to install {{site.prodname}} in 
-  > {{site.prodname}} CNI mode at present.
+  kubectl create -f {{ "/manifests/tigera-operator.yaml" | absolute_url }}
+  ```
+
+  > **Note**: Do not use the manifests from the [<u>current EKS docs</u>](https://docs.aws.amazon.com/eks/latest/userguide/calico.html)
+  > because they reference an older version of Calico, which does not support eBPF mode.
```

Comment:
This statement is likely to go out of date at some point.  Maybe:
```suggestion
  > because they reference an older version of Calico, which does not have the latest eBPF fixes.
```